### DISCUSSION TITLE
Need help to understand hammerDB

### DISCUSSION THREAD

**Discussion by Zeltron** [TPC-Council/HammerDB#88] [Q&A] [answered]
Hi there,

I'm new with hammerDB , I just have few questions. Is it hammerDB only work with 'tpcc' database ? can I use another database?

I'm trying to understand hammerDB

Thank you.

**Comment by sm-shaw** [comment, marked as answer]
The best place to start is at the HammerDB website https://www.hammerdb.com/ and the documentation starting with the Quick Start https://www.hammerdb.com/docs/ch02s01.html. 'tpcc' is not a database in itself it is a workload from the TPC http://www.tpc.org/ designed to benchmark database performance. The built-in HammerDB workloads are derived from TPC workloads (TPC-C for OLTP and TPC-H for OLAP) however you can write your own workloads as well if desired. The databases supported by HammerDB you can see from the Quick Start page are Oracle, Microsoft SQL Server, IBM Db2, MySQL (MariaDB), PostgreSQL and Redis. Therefore you would choose and install a database and then follow the Quick Start guide to run your first workload against the database you have chosen.

**Comment by Zeltron** [comment]
@sm-shaw  do you have any video demo on how to use hammerdb?

**Comment by sm-shaw** [comment]
HammerDB has not produced any video tutorials, all of the information is in the documentation and the blog. Having said that there are a number of third party videos available on the usual sites that can be easily found. Nevertheless the best way to learn is to start using it, HammerDB is a tool to test databases and therefore a lot of the knowledge required to get the best performance is in tuning the database side.
